<launch>
<executable cmd="ros2 bag play --clock 200 -r 1 $(find-pkg-share rerun_bridge)/go2_ros2" />

<!-- Run the Rerun bridge node -->
<node name="rerun_bridge_node" pkg="rerun_bridge" exec="visualizer">
    <param name="yaml_path" value="$(find-pkg-share rerun_bridge)/launch/go2_example_params.yaml" />
    <param name="use_sim_time" value="true" />
</node>
</launch>
